### ISSUE TITLE
[Bug Report] build dist (vuetify.min.js) is not customizable using Vue.use(Vuetify, {}) because already used in bundle

### ISSUE THREAD

**Issue by m-rod10** [vuetifyjs/vuetify#7791]
### Versions and Environment
**Vuetify:** 1.5.16
**Vue:** 2.6.9
**Browsers:** Firefox 69.0
**OS:** Windows 10

### Steps to reproduce
use buid dist in html with script src /node_modules/vuetify/dist/vuetify.min.js

then try to customize Vuetify

-script type="module"-
import vuetifyLocalefr from '/node_modules/vuetify/lib/locale/fr.js';
       
        Vue.use(Vuetify, {

            lang: { 
               locales: { fr: vuetifyLocalefr },
                current: 'fr'
            },
            
            theme:{
                primary:   "#4DB6AC",
                secondary: "#B2DFDB",
                accent: "#C2185B",
                error: "#E57373",
                warning: "#FFE0B2",
                info: "#90CAF9",
                success: "#81C784"
             }
        });

-/script-

### Expected Behavior
We should be able to use Vue.use(Vuetify,{}); once to configure the components

### Actual Behavior
As the dist include Vue.use(Vuetify, {}); we can't "reconfig" Vuetify because we can call only once Vue.use with the same id.

### Reproduction Link
<a href="https://github.com/nolink/nolink" target="_blank">https://github.com/nolink/nolink</a>

### Other comments
We use vuetify with native es6 module import and are not using builders like webpack

With regards,

**Comment by jacekkarczmarczyk** [comment]
```js
this.$vuetify.theme.primiary = '#123456';
this.$vuetify.lang.current = 'jp';
```

**Comment by KaelWD** [comment]
Also resolved in 2.0

**Comment by m-rod10** [comment]
thanks !
+
